### DISCUSSION TITLE
Runew, you have been rude to me by blocking me on Discord. People shouldn't be treated this way.  Now I have no respect for you or your work.  As a result, I won't be donating anymore.

### DISCUSSION THREAD

**Discussion by Amit30swgoh** [runew0lf/RuinedFooocus#119] [General]
Runew, you have been rude to me by blocking me on Discord. People shouldn't be treated this way. 
Now I have no respect for you or your work. 
As a result, I won't be donating anymore.

**Comment by runew0lf** [comment]
you never donated anyway, and were in the wrong server, and constantly dm'ing without permission
